.azure-pipelines/templates/windowsBuild.yml

```
@@ -16,10 +16,12 @@ steps:
       versionSpec: '3.x'
   - script: |
       choco install -y antlr4
```

```suggestion
      choco install -y antlr4
```
Not used by JV2. There might be refs to ANTLR4 in the other pipelines that can also be removed.